projects/plugins/social/src/js/components/info-section/index.js

```diff
@@ -1,33 +1,48 @@
-import { Container, Text } from '@automattic/jetpack-components';
+import { Container, Text, useBreakpointMatch } from '@automattic/jetpack-components';
 import { __ } from '@wordpress/i18n';
+import classNames from 'classnames';
 import styles from './styles.module.scss';
 
-const InfoSection = () => (
-	<Container horizontalSpacing={ 7 } horizontalGap={ 3 }>
-		<div className={ styles.column }>
-			<Text variant="title-medium" className={ styles.title }>
-				{ __( 'Did you know?', 'jetpack-social' ) }
-			</Text>
-			<Text variant="headline-small-regular" component={ 'span' } className={ styles.number }>
-				40x
-			</Text>
-			<Text>
-				{ __(
-					'Visual content is 40 times more likely to get shared on social media than any other type. Remember to include an image.',
-					'jetpack-social'
-				) }
-			</Text>
-			<Text variant="headline-small-regular" component={ 'span' } className={ styles.number }>
-				10x
-			</Text>
-			<Text>
-				{ __(
-					'By publishing at least once per week, you’ll be ahead of 99% of all other sites. Promoting that weekly content on social media may grow your audience by 10x in a few short months.',
-					'jetpack-social'
-				) }
-			</Text>
-		</div>
-	</Container>
-);
+const InfoSection = () => {
+	const [ isLg ] = useBreakpointMatch( 'lg' );
+	const [ isAtLeastMedium ] = useBreakpointMatch( 'md', '>=' );
+
+	const viewportClasses = {
+		[ styles[ 'is-viewport-large' ] ]: isLg,
+		[ styles[ 'is-viewport-medium' ] ]: isAtLeastMedium,
+	};
+
+	return (
+		<Container horizontalSpacing={ 7 } horizontalGap={ 3 }>
+			<div className={ classNames( styles.column, viewportClasses ) }>
```

Comment:
I'm wondering if we should just add these classes to the entire Jetpack Social admin page? It seems redundant to add them this many times to different components. What do you think?

Reply:
Good idea! Could not move to AdminPage because of obfuscation, but I moved it to the top-most component.